public/pages/CreateTrigger/components/Action/actions/Message.js

```diff
@@ -194,6 +195,8 @@ export default function Message(
     _.set(values, actionableAlertsSelectionsPath, actionableAlertsSelections);
   }
 
+  if (!displayThrottlingSettings) _.set(values, `${actionPath}.throttle_enabled`, false);
```

Comment:
This disables throttling if the throttling configurations are not displayed.

Reply:
This looks good to me after exploring the file.